src/modules/launcher/Wox.Infrastructure/Storage/StorageHelper.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+using System.Windows.Markup;
+
+namespace Wox.Infrastructure.Storage
+{
+    public class StorageHelper
```

Comment:
Maybe we could be more specific with the name here?  Perhaps `StoragePowetoysVersionInfo`